frontend/src/modules/modals/MultiStepWizard.tsx

```diff
@@ -549,11 +495,11 @@ const MultiStepWizard = (): JSX.Element => {
    *
    */
   const getAlarmValues = (stateKey: string) => {
-    const param = multiParams.find((param: Data) => param.stateKey === stateKey);
+    const param = multiParams.find((param: InternalState) => param.stateKey === stateKey);
     if (param) {
-      if (param.alarmValues.length) return param.alarmValues;
+      if (param.alarmValues) return param.alarmValues;
     }
-    return [];
+    return { lower: 0, upper: 0 };
```

Comment:
I think this is another example where `null` is better as a sentinel value indicating that alarm limits are missing.